web/src/state_data.ts

```diff
@@ -126,6 +126,15 @@ export const server_emoji_schema = z.object({
 
 export const realm_emoji_map_schema = z.record(server_emoji_schema);
 
+export const anonymous_group_type = z.object({
+    direct_subgroups: z.array(z.number()),
+    direct_members: z.array(z.number()),
+});
+
+export const group_setting_type_schema = z.union([z.number(), anonymous_group_type]);
+
+export type GroupSettingType = z.infer<typeof group_setting_type_schema>;
```

Comment:
Don’t name a type using `Type` unless it’s a type *of a type*.

For example, `type MessageType = "stream" | "private"` is the type of message (recipient) types.

These are not types of types. They should be named `anonymous_group_schema`, `group_setting_value_schema`, `GroupSettingValue`.

Reply:
Sure. Will rename them in further PRs.